### DISCUSSION TITLE
Content Security Policy issue in console

### DISCUSSION THREAD

**Discussion by P-Oliomonte** [neuefische/questions#854] [Web] [answered]
Hello all, 

we have the following issue shown in in the console: 

Content Security Policy of your site blocks the use of 'eval' in JavaScript

<img width="261" alt="Bildschirmfoto 2024-08-13 um 11 48 47" src="https://github.com/user-attachments/assets/1df730b6-86d1-4667-b373-49b7cb0fad86">

Does anyone know what this is about? We suspect it has something to do with global styling (/styles/globals.css). 

[Link to PR](https://github.com/eneben/capstone-flip-wise/pull/8)

Thanks in advance!

**Comment by Roland-Hufnagel** [comment]
Since you work with styled-component you should define your global styles in a `styles.js` file. Please have a look at the handout of the styled-components session how to create global styles. You deleted that file and instead you created a `styles.css` instead. That's not compatible with styled-components.

**Comment by P-Oliomonte** [comment]
Thank you very much for your fast reply @Roland-Hufnagel, 

we changed the code like you suggested but still run into the same issue 🤔

**Reply by ahohnsen** [reply, marked as answer]
I cloned your repository and cannot reproduce the error. 🤔 
Are you the only one in your team getting this error or are all of you running into this issue?
I suggest to run your app in different browsers and on different computers. It could be that a chrome extension is responsible for that error.

**Comment by P-Oliomonte** [comment]
We didn't finally resolve the issue yet but it only shows on one of our computers and only in Chrome. It might have to do with a particular browser extension.
